docs/general/use-cases/bytecode-interpreters.md

```diff
@@ -0,0 +1,16 @@
+---
+sidebar_position: 2
+description: Generating machine code from bytecode with SLJIT.
+---
+
+# Bytecode Interpreters
+
+SLJIT's approach is very effective for bytecode interpreters, since their machine-independent bytecode (middle-level representation) typically contains instructions which either can be easly translated to machine code, or which are not worth to translate to machine code in the first place.
```

Comment:
When I wrote that part, I just added some words, but never be happy about it. If you have a better idea about explaining this, it could be better. Or perhaps we could delete the whole thing.